rednotebook/gui/main_window.py

```diff
@@ -666,6 +666,14 @@ def load_values_from_config(self):
         )
 
         self.set_font(config.read("mainFont", editor.DEFAULT_FONT))
+        
+        self.set_autoIndent()
+        
+    def set_autoIndent(self):
+        if self.journal.config.read("autoindent") == 1:
+            self.day_text_field.day_text_view.set_auto_indent(True)
+        else:
+            self.day_text_field.day_text_view.set_auto_indent(False)
```

Comment:
Let's shorten this a bit:

    def set_auto_indent(self):
        auto_indent = self.journal.config.read("autoindent") == 1
        self.day_text_field.day_text_view.set_auto_indent(auto_indent)